### DISCUSSION TITLE
How can I change the default docker IP range in Colima?

### DISCUSSION THREAD

**Discussion by ChrisLane** [abiosoft/colima#388] [Q&A] [answered]
The default Docker subnet range of `172.17.0.0/16` is conflicting with some of my other networks and so I would like to change it.
In a standalone Docker environment, this would be done by updating the `bip` value in `/etc/docker/daemon.json`.
How can I do this with Colima?

## Versions
Colima Version: 0.4.4
Lima Version: 0.11.3
Qemu Version: 7.0.0

## OS
macOS M1

**Comment by abiosoft** [comment, marked as answer]
Have you tried this?

https://github.com/abiosoft/colima/blob/main/docs/FAQ.md#how-to-customize-docker-config-eg-add-insecure-registries

**Reply by ChrisLane** [reply]
Thanks, I hadn't seen that and it certainly seems like what I need.

My config is now:
```
docker:
  bip: 172.26.0.1/16
```
